api/v1/server/handlers/v1/webhooks/receive.go

```diff
@@ -99,16 +100,76 @@ func (w *V1WebhooksService) V1WebhookReceive(ctx echo.Context, request gen.V1Web
 	payloadMap := make(map[string]interface{})
 
 	if rawBody != nil {
-		err := json.Unmarshal(rawBody, &payloadMap)
+		contentType := ctx.Request().Header.Get("Content-Type")
 
-		if err != nil {
-			return gen.V1WebhookReceive400JSONResponse{
-				Errors: []gen.APIError{
-					{
-						Description: fmt.Sprintf("failed to unmarshal request body: %v", err),
+		if strings.Contains(contentType, "application/x-www-form-urlencoded") {
+			/* Parse form-encoded data */
+			formData, err := url.ParseQuery(string(rawBody))
+			if err != nil {
+				return gen.V1WebhookReceive400JSONResponse{
+					Errors: []gen.APIError{
+						{
+							Description: fmt.Sprintf("failed to parse form data: %v", err),
+						},
 					},
-				},
-			}, nil
+				}, nil
+			}
+
+			/* Slack interactive payloads use a 'payload' parameter containing JSON
+			 * See: https://docs.slack.dev/interactivity/handling-user-interaction/#payloads
+			 * For GENERIC webhooks, we convert all form fields directly to the payload map
+			 */
+			if webhook.SourceName == sqlcv1.V1IncomingWebhookSourceNameSLACK {
+				payloadValue := formData.Get("payload")
+				if payloadValue == "" {
+					return gen.V1WebhookReceive400JSONResponse{
+						Errors: []gen.APIError{
+							{
+								Description: "missing payload parameter in form-encoded request",
+							},
+						},
+					}, nil
+				}
+				/* Parse the payload parameter as JSON */
```

Comment:
nit: I personally don't love comments like this 😅 not sure where you fall on this, but IMO this sort of comment is clear from reading the code